ssg/jinja.py

```diff
@@ -129,6 +129,15 @@ def process_file(filepath, substitutions_dict):
     return template.render(substitutions_dict)
 
 
+def process_string(string, substitutions_dict):
+    """
+    Process the jinja string. Return the result as a string. Note that this will not
+    load the project macros; use process_file_with_macros(...) for that.
```

Comment:
use ~`process_file_with_macros(...)`~  `process_string_with_macros(...)` for that.